examples/parse-text-from-any-block-type/index.js

```diff
@@ -0,0 +1,148 @@
+import { Client } from "@notionhq/client"
+import { config } from "dotenv"
+
+config()
+
+const pageId = process.env.NOTION_PAGE_ID
+const apiKey = process.env.NOTION_API_KEY
+
+const notion = new Client({ auth: apiKey })
+
+/* 
+---------------------------------------------------------------------------
+*/
+
+// Take rich text array from a block child that supports rich text and return the plain text.
+// Note: All rich text objects include a plain_text field.
+const getPlainTextFromRichText = richText => {
+  return richText.map(t => t.plain_text).join(", ")
+  // Note: A page mention will return "Undefined" as the page name if the page has not been shared with the integration. See: https://developers.notion.com/reference/block#mention
+}
+
+// Use the source URL and optional caption from media blocks (file, video, etc.)
+const getMediaSourceText = block => {
+  let source, caption
+
+  if (block[block.type].external) {
+    source = block[block.type].external.url
+  } else if (block[block.type].file) {
+    source = block[block.type].file.url
+  } else if (block[block.type].url) {
+    source = block[block.type].url
+  } else {
+    source = "Missing case for media block types: " + block.type
```

Comment:
If we aren't going to throw an error here, maybe we should surround with square brackets or something to make it a bit more indicative that it's not user text?